server/routerlicious/packages/services-core/src/emptyTaskMessageSender.ts

```diff
@@ -5,6 +5,10 @@
 
 import { ITaskMessage, ITaskMessageSender } from "./taskMessages";
 
+/**
+ * @deprecated - This was functionality related to RabbitMq which is not used anymore,
```

Comment:
```suggestion
 * @deprecated This was functionality related to RabbitMq which is not used anymore,
```

No dashes for deprecations I think. @Josmithr always has to remind me about this. Maybe it's finally sinking in! 😂

Reply:
Ah, yes, only params... copilot lied to me 😢